zerver/models.py

```diff
@@ -1428,6 +1428,11 @@ class UserProfile(AbstractBaseUser, PermissionsMixin):
 
     zoom_token: Optional[object] = models.JSONField(default=None, null=True)
 
+    # Contains a external uid of the user to make it able
+    # to sync mails form e.g. LDAP
+    # Contains JSON in the form of {"ldap": "TestUser"}
+    ext_auth_uid: Any = models.JSONField(default=dict)  # TODO type
```

Comment:
For a variety of reasons (including having a clear schema), I'd prefer to have this be in a separate database table, rather than using JSONField.

Reply:
Do I understand this/django correctly that I would create a Model with an LDAP property for that?